### ISSUE TITLE
Not able to achieve the same accuracy as claimed

### ISSUE THREAD

**Issue by asad0938** [khurramjaved96/mrcl#8]
Hi,
I evaluated the pre-trained model provided by you. I got the following results.
10 - 0.8467
50 - 0.6677
75 - 0.5864
100 - 0.5251
150 - 0.4444
200 - 0.3642
These numbers are less than that of the numbers for every class size provided in the revised paper. In the older version too the accuracies for 150 and 200 are not matching. You have mentioned that you have not released the code for the latest version. Could you please provide the results for the current implementation?

**Comment by khurramjaved96** [comment]
Hi, 
I've uploaded a version of the model trained with updated code in the google drive folder with the name "omniglot_oml.model."

You should be able to get results comparable to those in the paper. 

You can evaluate the model using: 
"python evaluate_omniglot.py --rln 6 --model omniglot_oml.model --name OML_test --runs 30 --test"

You should be able to get similar results if you meta-trained used existing code as well as long as you implement some form of regularization, such as early stopping on a validation set.
